tutorials/java/CurrentPlaceDetailsOnMap/README.md

```diff
@@ -7,7 +7,7 @@ This sample goes hand in hand with a tutorial for the Google Maps Android API:
 Prerequisites
 --------------
 
-- Android SDK v33
+- Android SDK v34
```

Comment:
We should list the minSDK requirement, not the targetSDK. This should say "Android API level 21+"

Reply:
Applies to all the READMEs